README.md

```diff
@@ -21,17 +21,76 @@ This Client works against versions 4 to 8 of the NGINX Plus API. The table below
 | 7 | R25 |
 | 8 | R27 |
 
-## Using the Client
+## Using the client library
+
+Import the library using:
+```go
+import "github.com/nginxinc/nginx-plus-go-client/client"
+```
+
+## Creating a client
+
+Create a new ```Client``` that works with the latest NGINX API Version:
+```go
+myHTTPClient := &http.Client{
+    Timeout: 5*time.Second
+}
+c, err := NewNginxClient(myHTTPClient, "your-api-endpoint")
+if err != nil {
+    // handle error
+}
+```
+
+Create a new ```Client``` with specified API version:
```

Comment:
Should we make a note that this will attempt to connect to the server straight away to check if the version is compatible? there is a difference in behaviour between this and the default client, which is not obvious to the consumer of the api.

Reply:
Added more information in the new commit